### DISCUSSION TITLE
How can I set a minimum value for the LineChart

### DISCUSSION THREAD

**Discussion by AbdulRIsmail** [Abhinandan-Kushwaha/react-native-gifted-charts#698] [General]
Hi,

How can we add a minimum value to the chart (ideally the min value of the array), since it starts at 0, it makes the chart look like a close line since the data is similar

![image](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/39646629/a8aa1a1c-c964-4c16-b219-d6e828a75222)

**Comment by Abhinandan-Kushwaha** [comment]
Hi @AbdulRIsmail 
You can use the `yAxisOffset` prop.
